### DISCUSSION TITLE
Not a crashing issue but get an error

### DISCUSSION THREAD

**Discussion by kcits** [MasaCMS/MasaCMS#109] [General]
https://github.com/MasaCMS/MasaCMS/blob/038af155ceeff7856478861e8b6d9f418cee4b4f/admin/assets/js/mura.js#L22562

For this line, It either needs to be updated to:
//# sourceMappingURL=mura.min.js.map
Or can the line be deleted?

**Comment by guustnieuwenhuis** [comment]
I've created an issue #117 for this.

**Comment by guustnieuwenhuis** [comment]
We've fixed this issue in release 7.3.8
